test/optimizations/localon/local-block.good

```diff
@@ -0,0 +1,2 @@
+{x = 5}
+{x = local-block.chpl:12: error: cannot access remote data in local block
```

Comment:
Oh wait, I guess you're only running it for comm!=none, so you don't need to suppress --fast